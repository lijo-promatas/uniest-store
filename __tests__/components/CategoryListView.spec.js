import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';

import CategoryListView from '../../src/components/CategoryListView';

it('renders <CategoryListView /> component correctly', () => {
  const tree = renderer.create(<CategoryListView
    category={{
      main_pair: {
        detailed: {
          http_image_path: 'http://test.com',
        }
      }
    }}
    onPress={() => {}}
  />).toJSON();
  expect(tree).toMatchSnapshot();
});
