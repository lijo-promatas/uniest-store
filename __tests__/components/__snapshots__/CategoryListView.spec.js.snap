// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders <CategoryListView /> component correctly 1`] = `
<View
  accessible={true}
  isTVSelectable={true}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "opacity": 1,
    }
  }
>
  <View>
    <View>
      <Text
        accessible={true}
        allowFontScaling={false}
        ellipsizeMode="tail"
        style={
          Array [
            Object {
              "color": undefined,
              "fontSize": 30,
            },
            undefined,
            Object {
              "fontFamily": "Material Icons",
              "fontStyle": "normal",
              "fontWeight": "normal",
            },
            Object {},
          ]
        }
      >
        î£‹
      </Text>
    </View>
    <View>
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        numberOfLines={2}
      />
    </View>
  </View>
</View>
`;
